h2o-hadoop/scripts/Jenkins/Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/groovy
+
+@Library('test-shared-library') _
+import ai.h2o.ci.Utils
+def utilsLib = new Utils()
+
+pipeline {
+  agent none
+
+  parameters {
+      string(name: 'gitBranch', defaultValue: 'mr/feature/dockers', description: 'H2O-3 branch to build')
+      string(name: 'version', description: 'CDH or HDP version')
+      choice(
+              choices: 'cdh\nhdp',
+              description: 'Hadoop Distribution',
+              name: 'distribution')
+  }
+
+  environment {
+      CREDENTIALS_ID = '0c10faaf-da0a-4a47-89b8-7aef9269a4fb'
+      GIT_URL = 'https://github.com/h2oai/h2o-3.git'
```

Comment:
THis is part of Jenkins settings, not necessary.

Reply:
removed